src/content/docs/style-guide/writing-guidelines/user-related-language-guidelines.mdx

```diff
@@ -90,32 +90,37 @@ We recommend linking to the [explanation of default groups](/docs/accounts/accou
 
 ## General philosophy for user permissions/roles language [#user-language-philosophy]
 
-Our general philosophy is summed up in these points:
+First, it will help you to understand the relationship between groups, roles, and capabilities in the new New Relic One user model (sometimes referred to as v2 users). We recommend reading [this explanation of how these concepts relate](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/#type-role-group). 
```

Comment:
This first sentence feels a little strange. What is the "it" in "it will help you to understand"?

Reply:
Also, I feel like it's worth making it more clear what "these concepts" are.

Reply:
In some ways, I prefer the simplicity of what you deleted over what you've added here....